x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/add_prebuilt_rules_header_buttons.tsx

```diff
@@ -21,12 +21,13 @@ export const AddPrebuiltRulesHeaderButtons = () => {
   const shouldDisplayInstallSelectedRulesButton = numberOfSelectedRules > 0;
 
   const isRuleInstalling = loadingRules.length > 0;
+  const isRequestInProgress = isRuleInstalling || shouldShowLinearProgress;
```

Comment:
Separating this into two flags, because I want to disable the button either when any rule is installing OR we are installing the package in the background (`isRequestInProgress` case), **but** only show the spinner when a rule is installing (`isRuleInstalling` case)